changelog.md

```diff
@@ -1,3 +1,10 @@
+3.4.0.1
+=======
+- @arthurxavierx
+  - [#221](https://github.com/bitemyapp/esqueleto/pull/221)
+    - Make `ToAliasT` an associated type family of the `ToAlias` type class
+    - Make `ToAliasReferenceT` an associated type family of the `ToAliasReference` type class
```

Comment:
i guess we gotta update this now

Just in the interest of backwards compat, might be nice to have a thing like `type ToAliasT a = a` and a `{-# DEPRECATED "This type alias doesn't do anything. Please delete it. Will be removed in the next release. #-}` pragma on it

Reply:
> i guess we gotta update this now

Ah yes, of course 😅 

> Just in the interest of backwards compat, might be nice to have a thing like type ToAliasT a = a and a {-# DEPRECATED "This type alias doesn't do anything. Please delete it. Will be removed in the next release. #-} pragma on it

Great idea! Adding it right now, thanks.